client/src/components/spotManagement/AddSpot.jsx

```diff
@@ -36,10 +39,9 @@ class AddSpot extends React.Component {
     })
     .then( async () => {
       console.log('success adding');
-      // maybe not necessary
       await this.setState({
```

Comment:
@carrjsn @raphaelwho Using await before this.setState doesnt make sense to me. If you do want something to run immediately after setState, why not use a callback?
https://iamsongcho.medium.com/is-setstate-async-b1947fbb25e5